python_workflow_definition/src/python_workflow_definition/shared.py

```diff
@@ -10,6 +12,11 @@ def get_kwargs(lst):
     return {t['targetHandle']: {'source': t['source'], 'sourceHandle': t['sourceHandle']} for t in lst}
 
 
+def get_item(data: dict, key: str):
```

Comment:
As far as I see this is only used in `xing-v2.py`, but no longer in the notebooks, correct? In that case it could be removed here.